scripts/check_contract.py

```diff
@@ -15,8 +19,8 @@ def main(
             print("Signature found")
         else:
             print("Signature not found")
-
-        if is_checksum_valid(adoc_file_path, expected_checksum):
+        
+        if get_pdf_checksum_from_adoc(adoc_file_path) == expected_checksum:
```

Comment:
```suggestion
        if expected_checksum  and get_pdf_checksum_from_adoc(adoc_file_path) == expected_checksum:
```
Just to be sure we are not comparing `None == None`